ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/LedgerBackedWriteService.scala

```diff
@@ -98,4 +98,6 @@ private[stores] final class LedgerBackedWriteService(ledger: Ledger, timeProvide
       submissionId: Ref.SubmissionId,
   ): CompletionStage[state.PruningResult] =
     CompletableFuture.completedFuture(state.PruningResult.NotPruned(Status.UNIMPLEMENTED))
+
+  override def isDeduplicationEnabled: Boolean = false
```

Comment:
I think we _do_ need to set this to `true` for Sandbox Classic, as it currently relies on that feature. We have upcoming work to fix this.